### PR TITLE
Simplify Rails Install

### PR DESCRIPTION
**Pull request by alxjrvs**
This PR: 
 - Removes `workflows` from `.gnarrc`, as they are available in the `.github` repo. we now pull directly from that. Maybe this should be error-handled in case this is changed in the future. 
- Removed Bin/pronto, bin/rspec, bin/rubocop. No need to `bin/` unecessarily. 
- *Removes Pronto, Rubocop, and Gnar-Style*. This is something of an ideological change; with the `pronto` workflow, we only need the configuration files. 

I am leaning towards `.gnarrc` eating `gnar-style`, as the pronto runners (which are dockerized) do not have access to `gnar-style`. Making it open face allows us to keep it up to date as well. 

We'll see!

### CODE REVIEW

File: rails/7/files/.rubocop.yml [hunk 1]
```diff
@@ -1,3 +1,81 @@
...
+AllCops:
+  NewCops: enable
+  Exclude:
+    - db/schema.rb
+    - 'bin/**/*'
+    - 'node_modules/**/*'
+    - 'vendor/**/*'
+
+require:
+  - rubocop-performance
```
Comment (nialbima) [inline rails/7/files/.rubocop.yml:10]: unless we've got a project that won't use it, i'd argue that `rubocop-rspec` should also be in here.

File: rails/7/gnarly.rb [hunk 2]
```diff
@@ -345,12 +336,12 @@ def post_install_instructions
...
 def format_ruby
-  run 'bin/rubocop -A --fail-level error'
+  run 'bundle exec rubocop -A --fail-level error'
 end
 
 def completion_notification
   puts ''
-  ascii_art
+  puts ascii_art
   post_install_instructions
```
Comment (nialbima) [inline rails/7/gnarly.rb]: I'd argue that post_install_instructions should be one big string that gets puts-ed here.

File: rails/7/gnarly.rb [hunk 3]
```diff
@@ -345,12 +336,12 @@ def post_install_instructions
 end
 
 def format_ruby
-  run 'bin/rubocop -A --fail-level error'
+  run 'bundle exec rubocop -A --fail-level error'
 end
 
 def completion_notification
   puts ''
```
Comment (nialbima) [inline rails/7/gnarly.rb]: Is this just so there's a newline? You could just put it at the start of your ascii_art and remove this.

File: rails/7/files/.rubocop.yml (hunk 1 above)
Comment (alxjrvs) [reply rails/7/files/.rubocop.yml:10]: Adding now

**Pull request closed without merging**